azure-pipelines/templates/ci-opengfx.yml

```diff
@@ -2,7 +2,7 @@ steps:
 - bash: |
     set -ex
     cd bin/baseset
-    curl -L https://binaries.openttd.org/extra/opengfx/0.5.2/opengfx-0.5.2-all.zip > opengfx-0.5.2-all.zip
-    unzip opengfx-0.5.2-all.zip
-    rm -f opengfx-0.5.2-all.zip
+    curl -L https://binaries.openttd.org/installer/opengfx-1.2.0.7z > opengfx.7z
```

Comment:
Please don't use the installer link. I really like only NSIS uses it, and I really hope we can get ride of that too. It is really annoying to maintain. Also, it is confusing for many. Please use the real OpenGFX version. Tnx :)